torch/_functorch/vmap.py

```diff
@@ -415,6 +515,9 @@ def chunk_vmap(
     everything) and map (which executes things sequentially). ``chunk_vmap`` vectorizes the input with number of
     chunks at a time. For more details about vectorizing map, see :func:`vmap`.
 
+    .. note::
+        It is recommended to use :func:`vmap` with :attr:`chunk_size` argument.
```

Comment:
nit: "Please use :func:`vmap` with :attr:`chunk_size` argument instead of this API" to make it clear it's a replacement

Reply:
Also - Don't use 
```
:attr:`chunk_size`
```
Use 
```
``chunk_size``
```
instead. 

PyTorch's usage of :attr: is wrong -- :attr: should be used for Object attributes, not arguments to functions. https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py-attr